Sources/AlamofireSessionRenewer/AlamofireSessionRenewer.swift

```diff
@@ -13,7 +13,12 @@ public typealias FailureRenewHandler = (Bool) -> Void
 public typealias RenewCredentialHandler = ((@escaping SuccessRenewHandler, @escaping FailureRenewHandler) -> Void)
 
 /// This class is responsible for authentication credentials renewing process
-open class AlamofireSessionRenewer: RequestInterceptor {
+open class AlamofireSessionRenewer: RequestInterceptor, @unchecked Sendable {
```

Comment:
Why these changes are needed? And why do you use lock for credentials?

Reply:
> Non-final class 'AlamofireSessionRenewer' cannot conform to 'Sendable'; use '@unchecked Sendable'; this is an error in the Swift 6 language mode

> Stored property '_credential' of 'Sendable'-conforming class 'AlamofireSessionRenewer' is mutable; this is an error in the Swift 6 language mode

We got these errors, so to fix the first one I added `@unchecked Sendable` mark.

And then since the `credentials` property is public, I made it thread safe

Reply:
But we haven't updated to the swift 6 yet. If these are just a warnings let's keep them and fix in the scope of adding swift concurrency support.